nsv13/code/modules/squads/squad_vendor.dm

```diff
@@ -32,13 +32,19 @@
 	resistance_flags = ACID_PROOF | FIRE_PROOF
```

Comment:
Could you make the name of the regular squad vendor written without capitalization? Just an extra request.